lib/profile/info.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+ 
+// TODO: Update this placeholder widget
+class Info extends StatelessWidget {  
+  const Info ({Key key}) : super(key: key);
+  
+   @override
```

Comment:
Another nit-picky comment about indentation between `@override` and `Widget build...`. I'll stop commenting on the indentation and let you do what you will with this info LOL.

Reply:
LOOOOOOL oops, my bad! I didn't notice it was indented, thanks for calling it out! I'll fix all of them

Reply:
No worries! And thanks! 😘